script/Arcueid/Day 10/10_00_ARC10_5B.txt

```diff
@@ -270,7 +270,7 @@ Even now, if you were to ask me if I'm enjoying life,
 [sha:81460888a5c3a2f0a0a0285a44e1df8cf895dd20]{
 -- Page 488, Offset 25523.
 -- どんな絶望でも、在るという事は満ち足りている。
-No matter how much despair I feel, I'm content simply existing.
+No matter how much despair I feel, I'm content by simply existing.
```

Comment:
I like the simplicity of the original a little bit more.

Reply:
Agree